test.js

```diff
@@ -121,6 +121,20 @@ test('.stopAndPersist() - with new symbol and text', macro, spinner => {
 	spinner.stopAndPersist({symbol: '@', text: 'all done'});
 }, /@ all done/);
 
+test('spinner with default indent (0)', t => {
+	const spinner = new Ora('foo');
+	t.falsy(spinner.indent);
+});
+
+test('spinner with indent (7)', t => {
```

Comment:
You need to understand what your changes in this PR do. `this.stream.cursorTo(this.indent);` doesn't indent the string with spaces, but instead moves the CLI cursor. So you need to check for the correct ansi escape codes and that they match the indent option value.